model/permission.go

```diff
@@ -641,6 +644,22 @@ func initializePermissions() {
 		PERMISSION_DEMOTE_TO_GUEST,
 		PERMISSION_USE_CHANNEL_MENTIONS,
 	}
+
+	CHANNEL_MODERATED_PERMISSIONS = []string{
+		PERMISSION_CREATE_POST.Id,
+		"create_reactions",
+		"manage_members",
+		PERMISSION_USE_CHANNEL_MENTIONS.Id,
+	}
+
```

Comment:
This list is required to maintain a specific order of permissions when responding with the api since maps are unordered